src/main/kotlin/com/quipalup/katydid/logentry/application/UpdateLogEntryByIdCommandHandler.kt

```diff
@@ -0,0 +1,22 @@
+package com.quipalup.katydid.logentry.application
+
+import arrow.core.Either
+import arrow.core.flatMap
+import arrow.core.right
+import com.quipalup.katydid.common.id.Id
+import com.quipalup.katydid.logentry.domain.LogEntryRepository
+import com.quipalup.katydid.logentry.domain.UpdateLogEntryError
+import com.quipalup.katydid.logentry.primaryadapter.httprest.LogEntryUpdateAttributes
+import java.util.UUID
+import javax.inject.Named
+
+@Named
+class UpdateLogEntryByIdCommandHandler(private val logEntryRepository: LogEntryRepository) {
+    fun execute(command: UpdateLogEntryByIdCommand): Either<UpdateLogEntryError, Id> =
+        Id(UUID.fromString(command.id)).right()
+            .flatMap {
```

Comment:
1. find entity by id -> LogEntry
2. Modify internal state of LogEntry // domain layer
3. save(logEntry)
https://www.loom.com/share/43d5d684975046a68e2281d2cc3bfe4a